projects/static/projects/js/members.js

```diff
@@ -0,0 +1,28 @@
+function resetRole(selector_id) {
+  var selector = document.getElementById(selector_id);
+  for (var j = 0; j < selector.options.length; j++) {
+    var opt = selector.options[j];
+    if (opt.disabled) {
+          selector.value = opt.value;
+          break;
+    }
+  }
+  selector.disabled = true;
+  document.getElementById("submit_" + selector.id).style.display = "none";
+  document.getElementById("cancel_" + selector.id).style.display = "none";
+}
+
+function enableChangeRole(email) {
+  var elem_id = "role_" + email;
+  var selectors = document.getElementsByClassName("role_selector");
+  for (var i = 0; i < selectors.length; i++) {
+    var selector = selectors[i];
+    if (selector.id == elem_id) {
+      selector.disabled = false;
+      document.getElementById("submit_" + selector.id).style.display = "inline";
```

Comment:
I think if you set it to "", it effectively resets it to its old value, which is a bit more friendly to the CSS on the page.